governance/polls/templates/MIPs/Budget Ratification Poll - $mip_code - $date_MONTH_DD,_YYYY.md

```diff
@@ -7,8 +7,8 @@ parameters:
     type: rank-free
     abstain: [0]
   victory_conditions:
-    - { 
```

Comment:
I'm a bit worried about this bit. @prose11 ?

Reply:
Good shout -

We have this tool from DUX for checking them out, so I'll take care of any irregularities 

https://vote.makerdao.com/polling/create#poll-markdown-checker